plugin.xml

```diff
@@ -14,12 +14,10 @@
           <param name="onload" value="true" />
       </feature>
     </config-file>
-    <header-file src="src/ios/CDVRemoteInjection.h" />
-    <source-file src="src/ios/CDVRemoteInjection.m" />
     <header-file src="src/ios/CDVRemoteInjectionWebViewBaseDelegate.h" />
```

Comment:
We might as well delete this abstraction for more clarity since we don't need to handle UIWebView anymore. What do you think?

Reply:
I think yes, but I don't have enough skills to merge insane iOS code